doc/nrf/releases/release-notes-latest.rst

```diff
@@ -209,8 +228,54 @@ The following list summarizes the most important changes inherited from upstream
       * ``LINE_CTRL_DCD``
       * ``LINE_CTRL_DSR``
 
+  * SPI
+
+    * Added support for SPI emulators
+
   * USB:
 
     * Fixed handling of zero-length packets (ZLP) in the Nordic Semiconductor USB Device Controller driver (usb_dc_nrfx).
     * Fixed initialization of the workqueue in the usb_dc_nrfx driver, to prevent fatal errors when the driver is reattached.
     * Fixed handling of the SUSPEND and RESUME events in the Bluetooth classes.
+
+* Build system
+
+  * We now ensure that shields can be placed in other BOARD_ROOTs
+  * Added Basic support for clang 10 with x86
+
+* Documentation
+
+  * Documentation is now generated from the DTS bindings
+
+* Other
+
+  * Added initial LoRaWAN support
+  * west: nrfjprog: Fail if hex file has UICR data and no --erase
+
+* Sys
+
+  * New cbprintf API that provides a <x>printf family of functions but with a callback on character output, to perform in-place streaming of the formatted string
+  * minimal libc: Print stderr just like stdout
+  * Ring buffer: Allow using the full buffer capacity instead of forcing an empty slot
+  * Added a new ``CLAMP`` macro
+  * libc: minimal: Implement ``abort()``
+  * tracing: Added a feature for post-mortem analysis
+
+* Samples
+
+  * led_ws2812: Force SPIM on nRF52 DK
+  * ieee802154: New serialization-based sample for nRF53
+  * net: Introduced TagoIO IoT Cloud HTTP post client
+  * net: Civetweb WebSocket Server sample
+  * display: cfb_custom_font: Added support for ssd1306fb
+  * net: gsm: Add suspend/resume shell commands
+
+* Logging
+
+  * Added STP transport and raw data output support for systrace
+
+* Modules
+
+  * mbedtls: Introduced MBEDTLS_MEMORY_DEBUG option
+  * lvgl: Updated LVGL to v7.6.1
+  * openamp: Updated libmetal and openamp to v2020.10
```

Comment:
I think it should be also mentioned that nrfx in hal_nordic was updated to version 2.4.0.

Reply:
Doing that @anangl thanks